esi_leap/common/idp/baseIDP.py

```diff
@@ -0,0 +1,15 @@
+import abc
+
+class BaseIDP(abc.ABC):
+  
+  def get_project_list():
```

Comment:
If this is an `abc.ABC`, then we should be decorating these methods with the [`abstractmethod` decorator](https://docs.python.org/3/library/abc.html#abc.abstractmethod).